altimeter.py

```diff
@@ -0,0 +1,74 @@
+from machine import I2C, Pin, SPI
+from time import sleep
+import uos
+import bmp280
+import sdcard
+
+# BMP280 initialisation (i2c)
+i2c = I2C(0, sda=Pin(0), scl=Pin(1), freq=400000)
+bmp = BMP280(i2c)
+
+# SD card initialisation and mounting (SPI)
+spi = SPI(0, baudrate=1000000, polarity=0, phase=0, sck=Pin(2), mosi=Pin(3), miso=Pin(4))
```

Comment:
- could you use the SPI 1 bus because the lora is using the SPI 0 bus, would prob blocked if in use by one slave device
- can increase baudrate to 4MHz for faster r/w